lib/active_model/serializer/caching.rb

```diff
@@ -89,6 +89,10 @@ def fragmented(serializer)
         # https://github.com/rails-api/active_model_serializers/pull/1249#issuecomment-146567837
         def cache(options = {})
           self._cache = ActiveModelSerializers.config.cache_store if ActiveModelSerializers.config.perform_caching
```

Comment:
The real problem is that calling this method should enable the cache via sentinel value, rather than setting the cache store dependent on an controller attribute.  Each serializer could still have its own cache_store (though I'm not sure how often that would be used, and it causes extra complexity here).